internal/node/node.bzl

```diff
@@ -34,53 +34,42 @@ def _write_loader_script(ctx):
         escaped = mn.replace("/", r"\/").replace(".", r"\.")
         mapping = r"{module_name: /^%s\b/, module_root: '%s'}" % (escaped, mr)
         module_mappings.append(mapping)
+  node_modules_roots = []
```

Comment:
does it really work to take two top-level directories and give a unioned view of them to node resolve?
I'm thinking things like
```
top_level_A/
  mod1/
    mod2v1/
top_level_B/
  mod2v2/
```

now if you `require(mod2)` will you get v1 or v2?

Reply:
Should write a test to verify it but I think that depends on where you require(mod2) from. If its done from within mod1 then node module resolution will start there (`parent.filename` in `node_loader.js`) and it will get v1 since it depends on it. If its from the user code then it will get v2 in this case if it checks `top_level_A` first and then `top_level_B` node_modules.

But that changes if the structure looks like this.

```
top_level_A/
  mod1/
  mod2v1/
top_level_B/
  mod2v2/
```

Then it will depends on which top level is checked first if the require is from the user's code.